utils/installer/install.sh

```diff
@@ -458,7 +458,7 @@ function setup_lvim() {
 function create_desktop_file() {
   OS="$(uname -s)"
   # TODO: Any other OSes that use desktop files?
-  [ "$OS" != "Linux" ] && return
+  ([ "$OS" != "Linux" ] || ! command -v xdg-desktop-menu &>/dev/null) && return
   echo "Creating desktop file"
 
   for d in "$LUNARVIM_BASE_DIR"/utils/desktop/*/; do
```

Comment:
please re-work this to use `xdg-desktop-menu` instead of manually copying the file(s)

I think this should work

```sh
xdg-desktop-menu install --novendor ./utils/desktop/lvim.desktop
```

too bad that `xdg-icon-resource` doesn't support svg🤦